docs/skopeo-copy.1.md

```diff
@@ -84,6 +84,10 @@ Existing signatures, if any, are preserved as well.
 
 **--dest-compress-level** _format_ Specifies the compression level to use.  The value is specific to the compression algorithm used, e.g. for zstd the accepted values are in the range 1-20 (inclusive), while for gzip it is 1-9 (inclusive).
 
+**--src-registry-token** _Bearer token_ for accessing the source registry
```

Comment:
Please add periods to the end for each of these new entries ala:
```suggestion
**--src-registry-token** _Bearer token_ for accessing the source registry.
```

Reply:
Thanks. Adding to some other sentences in the doc that were missing the ending period too (as just copied-paste on of them as an example entry)